test/package.json

```diff
@@ -4,15 +4,13 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "test": "mocha --timeout 210000 --exit"
+    "test": "mocha --timeout 300000 --exit"
```

Comment:
😭 keeps getting worse lol

Reply:
I may be able to revert this. Was tweaking a bunch of settings while troubleshooting.